client/landing/stepper/utils/flow-redirect-handler.ts

```diff
@@ -0,0 +1,47 @@
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+
+// Flows to redirect
+const redirectRoutes = [
+	{ flow: '/setup/blog', to: '/start:lang?' },
+	{ flow: '/setup/free', to: '/start/free:lang?' },
```

Comment:
This is capturing the `/setup/free-post-setup` flow which AFAIK we're not removing

Reply:
This should be working now.